### DISCUSSION TITLE
可以配置内存当缓存来达到提高中转的速度吗？

### DISCUSSION THREAD

**Discussion by rakaki** [AlistGo/alist#1559] [Q&A]
我用的是玩客云刷armbian，使用夸克速度只有12m/s，我觉得可能是自带的存储限制了中转的速度。是否有办法让alist使用内存当缓存来提高中转的读写速度？

**Comment by rakaki** [comment]
alist放到机械硬盘试了下速度，没改变，看来瓶颈不在这里

**Reply by shaojs321** [reply]
这和你部署的vps上传网速有关
